#!/usr/bin/env python
PACKAGE = "aero_path_planning"

import roslib;roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()

gen.add("unkown_weight",    double_t, 0, "Weighting factor for unexpolored terrain", .1, 0, 1)
gen.add("difficult_weight", double_t, 0, "Weighting factor for difficult terrain",   .1, 0, 1)
gen.add("goal_weight",      double_t, 0, "Weighting factor for the goal",            .5, 0, 1)
gen.add("traversed_weight", double_t, 0, "Weighting factor for traversed terrain",   .2, 0, 1)

exit(gen.generate(PACKAGE, "aero_path_planning", "LocalPlanner"))